docs/data-sources/opsworkscm_server.md

```diff
@@ -1,64 +0,0 @@
----
```

Comment:
this file is being deleted so there was probably an issue with downloading the schema. We should suppress this one